### DISCUSSION TITLE
Which extensions are installed by default?

### DISCUSSION THREAD

**Discussion by ohhai-kthxbai** [hassio-addons/addon-vscode#259] [Q&A] [answered]
> The add-on has the Home Assistant, MDI icons and YAML extensions pre-installed and pre-configured right out of the box.

Which extensions are these please?
When I to view>extensions it shows that none are installed.
I really like them, and want to install them in a separate instance of vscode that I have.

Thanks!

**Comment by frenck** [comment, marked as answer]
You can find them here:

<https://github.com/hassio-addons/addon-vscode/blob/main/vscode/vscode.extensions>

They don't appear, as they are not user-installed extensions but installed/set as built-in extensions.

**Reply by ohhai-kthxbai** [reply]
Great, thank you!
